services/ui-src/src/constants.ts

```diff
@@ -1,2 +1,6 @@
 // BANNERS
 export const bannerId = "admin-banner-id";
+
+// REGEX
+export const dateFormatRegex =
+  /^((0[1-9]|1[0-2])\/(0[1-9]|1\d|2\d|3[01])\/(19|20)\d{2})|((0[1-9]|1[0-2])(0[1-9]|1\d|2\d|3[01])(19|20)\d{2})$/;
```

Comment:
Just moved to constants so we can reference it later. _Excellent_ regex-ing my man. 👍